src/InvokePlaster.ps1

```diff
@@ -830,7 +830,7 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
-                if (![string]::IsNullOrWhiteSpace($psVersion)) {
+                if (![string]::IsNullOrWhiteSpace($PowerShellVersion)) {
```

Comment:
done.. Sorry!! I'm a real screw up with this kind of casing ;) I'm used to CamelCase as you can see